### PR TITLE
Language inequality

### PR DESCRIPTION
**Pull request by averma1010**

### CODE REVIEW

**Review by GeoffreyXue — CHANGES_REQUESTED** [review]

File: content/topics/reduced-inequalities/Stories/Image_folder/saved.jpeg
Comment (GeoffreyXue) [inline content/topics/reduced-inequalities/Stories/Image_folder/saved.jpeg:1]: Is this image still needed? I see it isn't referenced anywhere in the story

File: content/topics/reduced-inequalities/Stories/Image_folder/Literacyrate.jpeg
Comment (GeoffreyXue) [inline content/topics/reduced-inequalities/Stories/Image_folder/Literacyrate.jpeg:1]: Same comment, is this needed if it isn't referenced?

**Review by GeoffreyXue — COMMENTED** [review]
Everything related to the story looks good! Just two questions on the remaining two images left in the story folder.

The deploy preview has the images too large, but that's a formatting issue on our side of the website

File: content/topics/reduced-inequalities/Stories/Image_folder/saved.jpeg
Comment (PranavC10) [reply content/topics/reduced-inequalities/Stories/Image_folder/saved.jpeg:1]: @averma1010 Hi Akshay Please could you check this.

Comment (averma1010) [reply content/topics/reduced-inequalities/Stories/Image_folder/saved.jpeg:1]: Hi,
The map shows the geographic distribution of the non-Spanish speakers.  I think it's important cause it gives you an idea that the states or municipalities that are affected are bordering each other and are not distributed randomly over the country. this coupled with the literacy map and their overlap hints at a systematic limitation or policy failure in these clusters of states.

File: content/topics/reduced-inequalities/Stories/Image_folder/Literacyrate.jpeg
Comment (averma1010) [reply content/topics/reduced-inequalities/Stories/Image_folder/Literacyrate.jpeg:1]: The paragraph below the maps references them in brief

File: content/topics/reduced-inequalities/Stories/Image_folder/saved.jpeg
Comment (averma1010) [reply content/topics/reduced-inequalities/Stories/Image_folder/saved.jpeg:1]: Oh, my bad. I didn't realize I had multiple map images. I have removed them now.

File: content/topics/reduced-inequalities/Stories/Image_folder/Literacyrate.jpeg
Comment (averma1010) [reply content/topics/reduced-inequalities/Stories/Image_folder/Literacyrate.jpeg:1]: Oh, my bad. I didn't realize I had multiple map images. I have removed them now.

**Review by GeoffreyXue — APPROVED** [review]
Looks good, thanks!

**Pull request merged**